docs/server-setup.md

```diff
@@ -66,10 +66,11 @@ To run your tests against a remote Selenium Server, you will need an account wit
 In your config file, set these options:
  - `sauceUser` -  The username for your Sauce Labs account.
  - `sauceKey` -  The key for your Sauce Labs account.
-
+ - `sauceSeleniumAddress` - default: ondemand.saucelabs.com:80/wd/hub ; travis: localhost:4445/wd/hub
+ 
 Please note that if you set `sauceUser` and `sauceKey`, the settings for `seleniumServerJar`, `seleniumPort` and `seleniumArgs` will be ignored.
 
-You can optionally set the [`name` property](referenceConf.js#L113) in a capability in order to give the jobs a name on the server.  Otherwise they will just be called `Unnamed Job`.
+You can optionally set the [`name` property](/docs/referenceConf.js#L113) in a capability in order to give the jobs a name on the server.  Otherwise they will just be called `Unnamed Job`.
```

Comment:
1. Not a compelling argument.  I'd rather have them both relative links.
2. Thanks for bringing this bug to my attention!  I'll fix it ASAP.  But changing the markdown because the website generation script doesn't work properly isn't really the right way approach these things.